shared/macros/10-bash.jinja

```diff
@@ -1862,9 +1865,9 @@ fi
 {{%- macro bash_ensure_pam_module_line(pam_file, group, control, module, after_match='') -%}}
 if ! grep -qP '^\s*{{{ group }}}\s+'"{{{ control }}}"'\s+{{{ module }}}\s*.*' "{{{ pam_file }}}"; then
     # Line matching group + control + module was not found. Check group + module.
-    if grep -qP '^\s*{{{ group }}}\s+.*\s+{{{ module }}}\s*' "{{{ pam_file }}}"; then
-        # If more occurrences match the search, only the first will be updated.
-        sed -i -E --follow-symlinks '0,/^\s*{{{ group }}}\s+.*{{{ module }}}\s*.*/ s/^(\s*{{{ group }}}\s+).*(\b{{{ module }}}.*)/\1'"{{{ control }}}"' \2/' "{{{ pam_file }}}"
+    if [ $(grep -qP '^\s*{{{ group }}}\s+.*\s+{{{ module }}}\s*' "{{{ pam_file }}}") -eq 1 ]; then
```

Comment:
Are you trying to count the number of lines matched?
If so, this won't work as you intend: https://www.gnu.org/software/grep/manual/grep.html#index-return-status

You,ll need to do `grep -cP` instead of `grep -qP`.

Reply:
Yes, I know. : ) This was the reason I set the `do-not-merge/work-in-progress` when my local tests, which were still running, showed this.

Reply:
@yuumasato , it should be ready for review now. Sorry for not making it more clear that I was still working on it before.